subtitles.go

```diff
@@ -927,3 +930,32 @@ func escapeHTML(i string) string {
 func unescapeHTML(i string) string {
 	return htmlUnescaper.Replace(i)
 }
+
+func NewScanner(i io.Reader) *bufio.Scanner {
```

Comment:
Unless I'm missing something, I don't see the point of exporting this method, therefore could you rename it to `newScanner` instead?

Reply:
Good point. Updated.